docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -74,3 +74,32 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
+
+## Add Dashboard and Datasource permissions to Service accounts
+
+<!-- Jo Guerreiro -->
+
+_Generally available in Grafana_
```

Comment:
So, dashboard permissions are available to OSS, Enterprise, Cloud. 
Datasource permissions are available for Enterprise, Cloud.

The ability to add service accounts to these permissions, everyone.
:sweat_smile: is there a better way to word it?

Reply:
Hmm, I think we should separate these entries then if they aren't available to the same set of users. Also, since these are both available to Cloud users, I'm going to add them to the Cloud what's new as well once we've edited them.